core/server/master/src/main/java/alluxio/master/file/PermissionChecker.java

```diff
@@ -127,375 +63,7 @@ void checkPermission(Mode.Bits bits, LockedInodePath inodePath)
-   * @param bits requested {@link Mode.Bits} by user
-   * @param path the path to check permission on
-   * @throws AccessControlException if permission checking fails
-   */
-  private void checkInode(String user, List<String> groups, Inode<?> inode, Mode.Bits bits,
-      String path) throws AccessControlException {
-    if (inode == null) {
-      return;
-    }
-    if (!mInodePermissionChecker.checkPermission(user, groups, inode, bits)) {
-      throw new AccessControlException(ExceptionMessage.PERMISSION_DENIED
-          .getMessage(toExceptionMessage(user, bits, path, inode)));
-    }
-  }
-
-  /**
-   * Gets the permission to access an inode path given a user and its groups.
-   *
-   * @param user the user
-   * @param groups the groups this user belongs to
-   * @param path the inode path
-   * @param inodeList the list of inodes in the path
-   * @return the permission
-   */
-  private Mode.Bits getPermissionInternal(String user, List<String> groups, String path,
-      List<Inode<?>> inodeList) {
-    int size = inodeList.size();
-    Preconditions
-        .checkArgument(size > 0, PreconditionMessage.EMPTY_FILE_INFO_LIST_FOR_PERMISSION_CHECK);
-
-    // bypass checking permission for super user or super group of Alluxio file system.
-    if (isPrivilegedUser(user, groups)) {
-      return Mode.Bits.ALL;
-    }
-
-    // traverses from root to the parent dir to all inodes included by this path are executable
-    for (int i = 0; i < size - 1; i++) {
-      try {
-        checkInode(user, groups, inodeList.get(i), Mode.Bits.EXECUTE, path);
-      } catch (AccessControlException e) {
-        return Mode.Bits.NONE;
-      }
-    }
-
-    Inode inode = inodeList.get(inodeList.size() - 1);
-    if (inode == null) {
-      return Mode.Bits.NONE;
-    }
-
-    return mInodePermissionChecker.getPermission(user, groups, inode);
-  }
-
-  private boolean isPrivilegedUser(String user, List<String> groups) {
-    return user.equals(mInodeTree.getRootUserName()) || groups.contains(mFileSystemSuperGroup);
-  }
-
-  private static String toExceptionMessage(String user, Mode.Bits bits, String path,
-      Inode<?> inode) {
-    StringBuilder sb =
-        new StringBuilder().append("user=").append(user).append(", ").append("access=").append(bits)
-            .append(", ").append("path=").append(path).append(": ").append("failed at ")
-            .append(inode.getName().equals("") ? "/" : inode.getName()).append(", inode owner=")
-            .append(inode.getOwner()).append(", inode group=").append(inode.getGroup())
-            .append(", inode mode=").append(new Mode(inode.getMode()).toString());
-    return sb.toString();
-  }
-=======
   void checkSetAttributePermission(LockedInodePath inodePath, boolean superuserRequired,
-      boolean ownerRequired) throws AccessControlException, InvalidPathException;
->>>>>>> master
+                                   boolean ownerRequired)
```

Comment:
(minor) I think the convention for line continuation is to do one extra indentation instead of align to the first parameter.

Reply:
done